src/genie/libs/parser/iosxe/show_license.py

```diff
@@ -178,3 +178,69 @@ def cli(self, output=None):
             ''' 
 
         return ret_dict
+
+# ----------------------
+# =================
+# Schema for:
+#  * 'show license summary'
+# =================
+class ShowLicenseSummarySchema(MetaParser):
+    """Schema for show license udi"""
+    schema = {
+        'license_usage': {
+            Any(): {
+                'entitlement': str,
+                'count': str,
+                'status': str,
+            }
+        }
+    }
+
+# =================
+# Parser for:
+#  * 'show license summary'
+# =================
+class ShowLicenseSummary(ShowLicenseSummarySchema):
+    """Parser for show license summary"""
+    cli_command = 'show license summary'
+    
+    """
+    License Usage:
+      License                 Entitlement Tag               Count Status
+      -----------------------------------------------------------------------------
+      network-advantage       (C9300-48 Network Advan...)       1 IN USE
+      dna-advantage           (C9300-48 DNA Advantage)          1 IN USE
+    """
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        # initial return dictionary
+        license_summ_dict = {}
+        
+        result_dict = {}
+
+        # license entitlement count status
```

Comment:
Please add example of what regex line is matching 
```suggestion
        # network-advantage       (C9300-48 Network Advan...)       1 IN USE
        # dna-advantage           (C9300-48 DNA Advantage)          1 IN USE
```